README.md

```diff
@@ -244,7 +244,7 @@ a Python version greater than 2.7 or 3.4 (Python <=3.3 is no longer supported).
 * **Cloudpickle**: Serialize variables in the IPython kernel to send to Spyder.
 * **spyder-kernels** 1.0+: Jupyter kernels for the Spyder console.
 * **keyring**: Save Github credentials to report errors securely.
-* **QDarkStyle**: A dark stylesheet for Qt applications, used for Spyder's dark theme.
+* **QDarkStyle** 2.6.0+: A dark stylesheet for Qt applications, used for Spyder's dark theme.
```

Comment:
We should require 2.6.3 IMO, since that's the first version with all the fixes for the major issues mentioned in #8068 . Otherwise, it won't look nearly as good as users expect and has several visible bugs and problems in our UI.